### DISCUSSION TITLE
Audio Panning for Individual Windows

### DISCUSSION THREAD

**Discussion by Macaw1975** [f1multiviewer/issue-tracker#184] [Feature requests]
Audio panning option per window. I'd use it when listening to multiple Team Radio. Being able to Pan Charles to the Right and Max to the Left would make distinguishing the 2 _much_ easier and "creates space" as we say in the audio world. With this you could pretty easily listen to 3 streams of audio without them stepping on each other auditorily.

Application could be just another Row in the audio setting section when right clicking on a stream window. A slider would work but I think it would be more annoying than functional and you'd have to apply a snap to middle function, I'd suggest this the below example as a drop down.

Pan :
L 100%, 
L 75%, 
L 50%,
L 25%,
C
R 25%
R 50%
R 75%
R 100%

**Comment by interi-mhickman** [comment]
I'd be happy with Left / Center / Right options that pan 100% left or right vs balanced. This is how teams listen to both their drivers' teams - one team in the left ear and one in the right.
